app/jobs/modsulator_job.rb

```diff
@@ -256,4 +258,14 @@ def status_ok(dor_object)
     status = dor_object.status_info[:status_code]
     [1, 6, 7, 8, 9].include?(status)
   end
+
+  private
+
+  def client
+    Faraday.new do |f|
+      f.request :multipart
+      f.request :url_encoded
+      f.adapter :net_http
```

Comment:
Can we look into using `http-client` here for better performance and persistent connections?
https://rubygems.org/gems/httpclient/
Compare with https://gist.github.com/darrenleeweber/89efe92d43449c07c729